src/components/shared/ProjectCard.tsx

```diff
@@ -72,8 +83,24 @@ export default function ProjectCard({
               {metadata.name}
             </h2>
 
-            <div style={{ color: colors.text.tertiary }}>
+            <div>
               <span style={{ color: colors.text.primary, fontWeight: 500 }}>
+                @{project.handle}
+              </span>
+              <span
+                style={{
+                  marginLeft: 10,
+                  color: colors.text.tertiary,
+                  fontSize: '0.7rem',
+                  fontWeight: 500,
+                }}
+              >
+                V{terminalVersion}
```

Comment:
**question**: should we use upper or lower-case v? (In the migrate modal, we're using lowercase)

Reply:
the uppercase looked better on the project cards and in the project dashboard, though lowercase feels more appropriate somehow. if we had to pick one to unify, i think my vote is uppercase

Reply:
I don't have a strong opinion - uppercase SGTM 💯